src/hub.html

```diff
@@ -397,27 +396,29 @@
                         position="0.275 0.175 0"
                         class="ui video-volume-up-button"
                     ></a-image>
-                    <a-image
-                        icon-button="image: #video-seek-back; hoverImage: #video-seek-back-hover;"
-                        scale="0.1 0.1 0.1"
-                        position="-0.3 -0.125 0"
-                        class="ui video-seek-back-button"
-                        visible="false"
-                    ></a-image>
-                    <a-image
-                        icon-button="image: #video-pause; hoverImage: #video-pause-hover; activeImage: #video-play; activeHoverImage: #video-play-hover"
-                        scale="0.15 0.15 0.15"
-                        position="0 -0.125 0"
-                        class="ui video-playpause-button"
-                        visible="false"
-                    ></a-image>
-                    <a-image
-                        icon-button="image: #video-seek-forward; hoverImage: #video-seek-forward-hover;"
-                        scale="0.1 0.1 0.1"
-                        position="0.3 -0.125 0"
-                        class="ui video-seek-forward-button"
-                        visible="false"
-                    ></a-image>
+                    <a-entity class="video-playback">
```

Comment:
Oof I hate to add another level here just for this. You already have a ref to back/forward/play.. Seems ok to just hide them directly.